rpc/jsonrpc/server/http_server_test.go

```diff
@@ -57,13 +56,10 @@ func TestMaxOpenConnections(t *testing.T) {
 			c := http.Client{Timeout: 3 * time.Second}
 			r, err := c.Get("http://" + l.Addr().String())
 			if err != nil {
-				t.Log(err)
```

Comment:
The other reporting methods, such as the variations of Log and Error, may be called simultaneously from multiple goroutines.